### PR TITLE
user id changed to user key

### PR DESCRIPTION
**Pull request by stopal-r7**

### CODE REVIEW

File: tests/test_team_api.py [hunk 1]
```diff
@@ -125,13 +125,13 @@ def test_add_user_to_team(mocked_url, mocked_rw_apikey, mocked_account_resource_
...
 
-    assert "Added user with id: '%s' to team\n" % user_id_to_add == out
+    assert "Added user with key: '%s' to team\n" % user_id_to_add == out
 
 @httpretty.activate
 @patch('lecli.apiutils.get_account_resource_id')
 @patch('lecli.apiutils.get_rw_apikey')
 @patch('lecli.team_api._url')
-def test_add_user_to_team(mocked_url, mocked_rw_apikey, mocked_account_resource_id, capsys):
+def test_delete_user_from_team(mocked_url, mocked_rw_apikey, mocked_account_resource_id, capsys):
```
Comment (eflanagan-r7) [inline tests/test_team_api.py:134]: Should there not be a `httpretty.register_uri(httpretty.DELETE...)` in this test instead of PUT?

**Comment by eflanagan-r7** [comment]
+1

File: tests/test_team_api.py (hunk 1 above)
Comment (stopal-r7) [inline tests/test_team_api.py:134]: Deleting a user from a team is supported with PUT at the moment. DELETE for teams API supports deleting a team only.

Comment (eflanagan-r7) [inline tests/test_team_api.py:134]: Is this because it's a soft delete?

**Pull request merged**